templates/migrate.yaml.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: "{{ k8s_namespace }}"
+  name: migrate
```

Comment:
I think this will create the `Job` once and then migrations will never run again. We might need to think through some way to delete this when it's done, or otherwise create a new job each time we attempt to run migrations. It might need to be its own task in the Ansible playbook, I'm not sure. 🤔